### DISCUSSION TITLE
Can I override certain components in the docs theme?

### DISCUSSION THREAD

**Discussion by sbvegan** [shuding/nextra#2491] [Q&A] [answered]
Is it possible to import custom components to override the built in components that come in the docs theme? For example: replacing the navbar

**Comment by dimaMachina** [comment, marked as answer]
https://nextra.site/docs/docs-theme/theme-configuration#customize-the-navbar

via `navbar.component`

**Reply by sbvegan** [reply]
Thank you, one follow up! If we want to customize the style. Would you suggest doing that on the component level with CSS?
